samples/ComputeSharp.SwapChain/Program.cs

```diff
@@ -35,28 +35,35 @@ class Program
 
     static void Main()
     {
-        Console.WriteLine("Available samples:");
-        Console.WriteLine();
+        int index;
 
-        for (int i = 0; i < Samples.Length; i++)
+        do
         {
-            Console.WriteLine($"{i}: {Samples[i].GetType().GenericTypeArguments[0].Name}");
-        }
+            Console.WriteLine("Available samples:");
+            Console.WriteLine();
 
-        Console.WriteLine();
+            for (int i = 0; i < Samples.Length; i++)
+            {
+                Console.WriteLine($"{i}: {Samples[i].GetType().GenericTypeArguments[0].Name}");
+            }
 
-        int index;
+            Console.WriteLine($"{Samples.Length}+: Exit (Use Escape, 'Q', or Alt+F4 to exit a sample once chosen.)");
+            Console.WriteLine();            
 
-        do
-        {
-            Console.Write("Enter the index of the sample to run: ");
-        }
-        while (!int.TryParse(Console.ReadLine(), out index));
+            do
+            {
+                Console.Write("Enter the index of the sample to run: ");
+            }
+            while (!int.TryParse(Console.ReadLine(), out index));
 
-        Console.WriteLine();
-        Console.WriteLine($"Starting {Samples[index].GetType().GenericTypeArguments[0].Name}...");
+            if (index >= 0 && index < Samples.Length)
+            {
+                Console.WriteLine();
+                Console.WriteLine($"Starting {Samples[index].GetType().GenericTypeArguments[0].Name}...");
 
-        Win32ApplicationRunner.Run(Samples[index]);
+                Win32ApplicationRunner.Run(Samples[index]);
+            }
+        } while (index >= 0 && index < Samples.Length);
```

Comment:
Nit: the `while` should go to a new line.
```suggestion
        }
        while (index >= 0 && index < Samples.Length);
```